setup.py

```diff
@@ -34,6 +34,7 @@ def read_requirements(path):
 setup(
     name="victron_ble",
     version=read("victron_ble", "VERSION"),
+    license="Unlicense",
```

Comment:
Be sure to also add the classifiers, it depends on your build system if it is able to automatically mark it as such